index.html

```diff
@@ -11,4 +11,3 @@
     <script type="module" src="/src/main.jsx"></script>
   </body>
 </html>
-<!-- Testing -->
```

Comment:
Leave an empty line at the end.

Reply:
this was old file. file has been changed now